getting-started-github-graphql-api.md

```diff
@@ -40,59 +42,59 @@ When interacting with GitHub's GraphQL API, it would be great to fill your own a
+In the next step, we add a new header with a name and value to your GraphiQL configuration. To communicate with GitHub's GraphQL API, fill in the header name with "Authorization" and the header value with "bearer [your personal access token]". Save this new header for your GraphiQL application. Finally, you are ready to make requests to GitHub's GraphQL API with your GraphiQL application.
 
 {{% pin_it_image "graphiql authorization" "img/posts/getting-started-github-graphql-api/graphiql-authorization.jpg" "is-src-set" %}}
 
-Last but not least, in case you end up using your own GraphiQL application, you need to provide the GraphQL endpoint for GitHub's GraphQL API which is `https://api.github.com/graphql`. In case of GitHub's GraphQL API, you are going to use the {{% a_blank "POST HTTP method" "https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods" %}} for GraphQL queries and mutations. After all, you have to transfer the data as payload to your GraphQL endpoint.
+If you use your own GraphiQL application, you'll need to provide the GraphQL endpoint for GitHub's GraphQL API:`https://api.github.com/graphql`. For GitHub's GraphQL API, use the {{% a_blank "POST HTTP method" "https://en.wikipedia.org/wiki/Hypertext_Transfer_Protocol#Request_methods" %}} for queries and mutations, and to transfer data as a payload to your GraphQL endpoint.
 
-In conclusion, the last paragraphs have provided you with two ways to interact with the GitHub GraphQL API. Whereas GitHub's GraphQL Explorer can only be used for GitHub's API, GraphiQL, integrated in your application or as standalone application, can be used for any GraphQL API. But it requires a bit more setup. As you might have noticed, the GitHub GraphQL Explorer is nothing else than a hosted standalone GraphiQL application, but it is tailored to only use GitHub's GraphQL API.
+This section provided you with two ways to interact with GitHub's GraphQL API. Where GitHub's GraphQL Explorer can only be used for GitHub's API, GraphiQL integrated into an application or standalone can be used for any GraphQL API. The difference is that it requires a bit more setup. The GitHub GraphQL Explorer is really nothing more than a hosted standalone GraphiQL application tailored to use GitHub's GraphQL API.
 
 <hr class="section-divider">
 
-After you went through these instructions that showed you how to set up GitHub in order to use their GraphQL API to learn about GraphQL itself, you should be ready to implement your first GraphQL client application with it. Follow me on this journey with your next GraphQL client-side application which we are going to build in React.
+After you've set up GitHub to use their GraphQL API to learn about GraphQL, you should be ready to implement your first GraphQL client application. Follow along and create your first (or your next) GraphQL client-side application in React.
```

Comment:
"Follow along and create your first (or your next) GraphQL client-side application in React." to "Follow along and create your first GraphQL client-side application in React."